manager/listen_other.go

```diff
@@ -0,0 +1,30 @@
+// +build !linux
+
+package manager
+
+import "net"
+
+// getDefaultListenIP attempts to get the default listen address by reaching out
+// to a public server and getting the local address
+func getDefaultListenIP() (string, error) {
+	conn, err := net.Dial("udp", "8.8.8.8:53")
+	if err != nil {
+		if e, ok := err.(net.Error); ok {
+			if e.Timeout() {
+				// Maybe it's ipv6 only?
+				var err2 error
+				conn, err2 = net.Dial("udp", "2001:4860:4860:0:0:0:0:8888")
```

Comment:
ah, you're probably right. I haven't setup a v6-only host yet. Will check this out.